pyclient/tests/test_query.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+from tests.testbase import BaseTestCase
+
+
+class QueryTestCase(BaseTestCase):
```

Comment:
This is only testing linear DAGs, so I can't see what the plan is for non-linear cases.